esp32_ulp/opcodes.py

```diff
@@ -378,9 +378,9 @@ def i_wait(cycles):
     return _delay.all
 
 
-def i_adc(reg_dest, adc_idx, pad_idx):
+def i_adc(reg_dest, adc_idx, mux):
     _adc.dreg = get_reg(reg_dest)
-    _adc.mux = get_imm(pad_idx) + 1
+    _adc.mux = get_imm(mux)
```

Comment:
i said "wrong in ulp.h".
of course it could also be "just different" semantics for the ADC macro than for the binutils assembler instruction.

Reply:
different semantics, i think. ulp.h takes the pad number as an argument, while in binutils the instruction take `mux` value, and these differ by 1.